packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -168,16 +171,18 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
       }
     }
 
+    const dataIsFocusable = ((this.props as any)['data-is-focusable'] === false || (disabled && !allowDisabledFocus) || this._isSplitButton) ? false : true;
+
     const buttonProps = assign(
```

Comment:
How does this work for buttons that ether contain menus or respond to keyEvents, won't the keyEvents still get executed when disabled? (pointer-events: none only stops the mouseDown, mouseUp, and click for firing)